YSI_Players/y_android/y_android_entry.inc

```diff
@@ -92,17 +92,25 @@ Optional plugins:
 
 static stock
 	YSI_g_sTimers[MAX_PLAYERS],
-	bool:YSI_g_sIsAndroid[MAX_PLAYERS] = bool:-1;
+	bool:YSI_g_sIsAndroid[MAX_PLAYERS] = {bool:-1, ...};
 
 forward OnAndroidCheck(playerid, bool:isDisgustingThiefToBeBanned);
 
 stock bool:IsPCPlayer(playerid)
 {
+	if (!IsPlayerConnected(playerid))
+	{
+		return bool:-1;
```

Comment:
This won't work properly.  Doing:

```c
if (IsPCPlayer(-1)) {}
```

Will return `true`.  Both functions should return `false` in the disconnected state.